pkg/templatelookup/regular.go

```diff
@@ -32,7 +30,9 @@ type ModuleTemplateInfo struct {
 	DesiredChannel string
 }
 
-func NewTemplateLookup(reader client.Reader, descriptorProvider *provider.CachedDescriptorProvider, syncEnabled bool) *TemplateLookup {
+func NewTemplateLookup(reader client.Reader, descriptorProvider *provider.CachedDescriptorProvider,
+	syncEnabled bool,
```

Comment:
`syncEnabled` can now also be removed from `TemplateLookup` 👍

Reply:
Nice, catch! Thanks